modules/next/src/Form/NextSettingsForm.php

```diff
@@ -43,8 +43,10 @@ class NextSettingsForm extends ConfigFormBase {
    */
   public function __construct(ConfigFactoryInterface $config_factory, SitePreviewerManagerInterface $site_previewer_manager, PreviewUrlGeneratorManagerInterface $preview_url_generator_manager = NULL) {
     if (!$preview_url_generator_manager) {
+      // phpcs:disable
       @trigger_error('Calling NextSettingsForm::__construct() without the $preview_url_generator_manager argument is deprecated in next:1.3.0. The $preview_url_generator_manager argument will be required in next:2.0.0. See https://www.drupal.org/project/next/releases/1.3.0', E_USER_DEPRECATED);
       $preview_url_generator_manager = \Drupal::service('plugin.manager.next.preview_url_generator');
```

Comment:
Use DI instead of calling static service.

Reply:
https://github.com/chapter-three/next-drupal/pull/313#discussion_r990094793 @chandan-singh7929